include/mfmg/dealii/dealii_matrix_free_operator.hpp

```diff
@@ -30,6 +36,14 @@ class DealIIMatrixFreeOperator : public DealIITrilinosMatrixOperator<VectorType>
 
   std::shared_ptr<Operator<VectorType>> multiply_transpose(
       std::shared_ptr<Operator<VectorType> const> b) const override final;
+
+  void vmult(vector_type &dst, vector_type const &src) const;
+
+  size_type m() const;
+
+  size_type n() const;
```

Comment:
@Rombur `n()` in not actually required to use `DealIIMatrixFreeOperator` as the `MatrixType` template argument of `dealii::PreconditionChebychev`.  Let me know if you'd like me to get rid of it.